ArcGISRuntimeSDKQt_CppSamples/Scenes/ScenePropertiesExpressions/ScenePropertiesExpressions.cpp

```diff
@@ -0,0 +1,146 @@
+// [WriteFile Name=ScenePropertiesExpressions, Category=Scenes]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "ScenePropertiesExpressions.h"
+
+#include "ArcGISTiledElevationSource.h"
+#include "Basemap.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+#include "SimpleMarkerSceneSymbol.h"
+#include "SimpleRenderer.h"
+
+using namespace Esri::ArcGISRuntime;
+
+namespace
+{
+  const QString HEADING("HEADING");
+  const QString PITCH("PITCH");
+}
+
+ScenePropertiesExpressions::ScenePropertiesExpressions(QObject* parent /* = nullptr */):
+  QObject(parent),
+  m_scene(new Scene(Basemap::imagery(this), this))
+{
+  // create a new elevation source from Terrain3D REST service
+  ArcGISTiledElevationSource* elevationSource = new ArcGISTiledElevationSource(
+        QUrl("https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer"), this);
+
+  // add the elevation source to the scene to display elevation
+  m_scene->baseSurface()->elevationSources()->append(elevationSource);
+
+  m_graphicsOverlay = new GraphicsOverlay(this);
+  m_graphicsOverlay->setSceneProperties(LayerSceneProperties(SurfacePlacement::Relative));
+
+  // Create a SimpleRenderer and set expressions on its scene properties:
+  // Renderer.getSceneProperties().setHeadingExpression("[HEADING]").
```

Comment:
I don't think you need this line - it's just duplicating the code?

Reply:
Done.